chaincfg/params.go

```diff
@@ -276,6 +280,11 @@ type Params struct {
 	// block height 1. If there are no payouts to be given, set this
 	// to an empty slice.
 	BlockOneLedger []*TokenPayout
+
+	// SuperMajorityMultiplier, SuperMajorityDivisor are used to calculate
+	// the super majority during version accounting.
+	SuperMajorityMultiplier int64
```

Comment:
I think these should be moved up with the other stake fields and named `StakeMajorityMultiplier` and `StakeMajorityDivisor` since there more than one "super majority" (the PoW version, PoS version, and eventually vote majority).